pipeline/git/git.go

```diff
@@ -30,5 +34,7 @@ func (Pipe) Run(ctx *context.Context) (err error) {
 		PreviousTag: previous,
 		Diff:        log,
 	}
+	// removes usual `v` prefix
+	ctx.Version = strings.TrimPrefix(tag, "v")
```

Comment:
I was checking and it seems that at least all popular Go projects follow the "vX.XX.XX" convention.
But I still think we shouldn't rely on it. If someone sets a tag name starting with "v" but not being of the same format, it might be a big surprise.
How about a quick check before trimming? A regex like "^v[0-9.]+" would do.